go-controller/pkg/allocator/id/allocator.go

```diff
@@ -26,7 +26,7 @@ type IDAllocator struct {
 
 // NewIDAllocator returns an IDAllocator
 func NewIDAllocator(name string, maxIds int) (*IDAllocator, error) {
-	idBitmap := bitmapallocator.NewContiguousAllocationMap(maxIds, name)
+	idBitmap := bitmapallocator.NewRoundRobinAllocationMap(maxIds, name)
```

Comment:
can this be squashed into the commit that introduced this code ? 

Like, having this as a separate commit raises the question of what changed, and I guess nothing did: you could have created the ID allocator with this strategy directly.

Reply:
I didn't introduce this code. I will add to the commit message the reason I did this change.